scripts/mysqld_safe.sh

```diff
@@ -957,7 +974,7 @@ then
   fi
 
   # Launch mysqld with numactl.
-  cmd="$cmd numactl --interleave=all"
+  [ $simulate -eq 0 ] && cmd="$cmd numactl --interleave=all"
```

Comment:
In mariadb-service-conver you anyway prepend this. Any reason to cut it off here?